### DISCUSSION TITLE
DataTable Horizontal scrolling while keeping header in line

### DISCUSSION THREAD

**Discussion by givihuda** [ColorlibHQ/AdminLTE#4742] [Q&A] [answered]
I'm not seeing how to do allow a DataTable to horizontal scroll should the column headers or values be too long for the box, so I added CSS style `overflow-x: scroll;` to:

`<div id="my-data-table_wrapper" class="dataTables_wrapper form-inline dt-bootstrap no-footer">`

and it horizontally scrolls everything fine until I vertically scroll too far, and then the sticky header (?) moves up to the top of the page but does not scroll horizontally with the data.

Here's an example with it just horizontally scrolled:

![image](https://user-images.githubusercontent.com/74785256/194941653-6c7fcce5-afa7-4c62-af1b-74f47f7b3783.png)

Here's with it after vertically scrolling:

![image](https://user-images.githubusercontent.com/74785256/194941682-d547a99b-444f-4c0b-b91f-63a1976e3743.png)

Notice how much they can become misaligned when both are scrolled.

Where is any of this even specified in docs?

**Comment by lenamtl** [comment, marked as answer]
Hi, 

This is not related to the template.
You can ask your question on Datatables forum https://datatables.net/forums/
